x/evm/keeper/keeper.go

```diff
@@ -51,12 +53,18 @@ type Keeper struct {
 	pendingTxs                   map[string][]*PendingTx
 	keyToNonce                   map[tmtypes.TxKey]*AddressNoncePair
 
-	// only used during ETH replay. Not used in chain critical path
+	// only used during ETH replay. Not used in chain critical path.
 	EthClient       *ethclient.Client
 	EthReplayConfig replay.Config
-	Trie            ethstate.Trie
-	DB              ethstate.Database
-	Root            common.Hash
+
+	// only used during blocktest. Not used in chain critical path.
+	EthBlockTestConfig blocktest.Config
```

Comment:
where are these fields set? You probably want to add it here https://github.com/sei-protocol/sei-chain/blob/seiv2/app/app.go#L593

Reply:
Enabling block test here https://github.com/sei-protocol/sei-chain/blob/709a84df53726609f50d991f747c85de07df9485/app/eth_replay.go#L103. Let me know if this should be removed.